### PR TITLE
Fixed Aggregate getTable and updated README.md

### PR DESCRIPTION
**Pull request by ianmahoney1**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,32 +25,34 @@ Installation steps:
...
-<br/> Here you have two tabs one for Aggregate and another for KoboToolbox
-Choose one of the tab and Enter url (required). Other fields are optional.
-<br/>
-You can create free account in <a href="https://www.kobotoolbox.org/">KoboToolbox server </a> and if you want to install your own aggregate server, <a href="http://docs.opendatakit.org/aggregate-guide/"> then visit </a>
-<br/>
-<img src="https://user-images.githubusercontent.com/5653512/67921427-aa90e200-fbcd-11e9-874b-fc1fae692fe0.png" alt="settings">
+<br/> Here you have three tabs one for Aggregate, KoboToolBox, and Central
+Choose one of the tabs and Enter url (required). Other fields are optional.
+<br/><br/>
+You can install your own aggregate server <a href="http://docs.opendatakit.org/aggregate-guide/"> here </a> <br/>
```
Comment (lognaturel) [inline README.md]: I recommend replacing this by Central and not linking to Aggregate docs. The plugin support will certainly still be helpful for folks who have existing Aggregate servers but we don't recommend setting up a new one.

A follow-on issue would be to swap the order of the Central and Aggregate tabs in the UI.

File: README.md [hunk 2]
```diff
@@ -25,32 +25,34 @@ Installation steps:
...
 <br/>
-If you are not sure how to create  value map in QGIS,<a href= "http://www.northrivergeographic.com/archives/qgis-and-value-maps"> Visit this link </a>
-<br/>
-The following video demonstrates, how to use the QRealTime Plugin:
+If you are not sure how to create  value map in QGIS, visit this <a href= "http://www.northrivergeographic.com/archives/qgis-and-value-maps"> link </a>
+<br/><br/>
+For a tutorial on how to use the QRealTime Plugin, check out this video:
 <br/>
-<iframe width="560" height="315" src="https://www.youtube.com/embed/zmr2CC5G-m4" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
+https://www.youtube.com/watch?v=62oqJE0pgIY
```
Comment (lognaturel) [inline README.md:55]: This is great! Make sure to change your password if you haven't already. 😊 A possible follow-up issue would be to show/hide that password.

File: README.md [hunk 3]
```diff
@@ -25,32 +25,34 @@ Installation steps:
...
 
+<h3>Using the Plugin:</h3>
 
 <br/>
 Right click over any existing layer --> QRealTime and choose desired option: 
-<br/>Make Online (To Create new form), import (to import data of existing form), sync(to automatically update your layer)
-<br/>If you want to import data from already existing form (created without using our plugin) do ensure that the name of the geometry field is 'GEOMETRY','location' or 'gps. Other geometry field names are currently not supported.
-<br/>
+<br/><br/>Make Online (to create new form), import (to import data of existing form), sync (to automatically update your layer)
+<br/><br/>If you want to import data from already existing form (created without using our plugin) do ensure that the name of the geometry field is 'GEOMETRY', 'location', or 'gps'. Other geometry field names are currently not supported.
```
Comment (shivareddyiirs) [inline README.md]: Constraint on name of the geometry is no more there. Please verify

**Review by shivareddyiirs — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (ianmahoney1) [reply README.md]: Yes this is true. Changed README to reflect this.

**Pull request merged**